service/resource/deployment/worker_deployment.go

```diff
@@ -54,6 +54,9 @@ func newWorkerDeployments(customObject kvmtpr.CustomObject) ([]*extensionsv1.Dep
 					Spec: apiv1.PodSpec{
 						Affinity:    newWorkerPodAfinity(customObject),
 						HostNetwork: true,
+						NodeSelector: map[string]string{
+							"role": WorkerID,
```

Comment:
what is this? what is `WorkerID` stands for?

In my PR we pin workers to nodes with labels `role=worker`

Reply:
`WorkerID` is just a constant with the value `worker`. See https://github.com/giantswarm/kvm-operator/blob/acc1ea5106690dc59ab73463013d3c7005fea0a6/service/resource/deployment/resource.go#L23.

Reply:
ok